packages/components/src/public/QueryModel/TabbedGridPanel.tsx

```diff
@@ -209,8 +223,10 @@ export const TabbedGridPanel: FC<TabbedGridPanelProps & InjectedQueryModels> = m
                     </ul>
                 )}
                 <GridPanel
+                    allowViewCustomization={allowViewCustomization}
                     key={activeId}
                     actions={actions}
+                    hasHeader={tabOrder.length === 1 && !alwaysShowTabs }
```

Comment:
looks like `tabOrder.length === 1 && !alwaysShowTabs` is used a couple times, make variable?

Reply:
Good idea. Done.